models/Program.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * custom type enumerating days of the week for the createProgramInput custom type
+ */
+export enum weekdays {
+    MON = "MON",
+    TUE = "TUE",
+    WED = "WED",
+    THU = "THU",
+    FRI = "FRI",
+    SAT = "SAT",
+    SUN = "SUN",
```

Comment:
do we need strings here?

Reply:
> Added a question for @feng-eric. Also, I think this PR does more than just the delete endpoint so consider changing the PR name.

It's because Brandon branched off of another local branch so there are changes that he didn't make. It's a bit confusing yeah